addons/website_sale/static/src/js/website_sale.js

```diff
@@ -163,6 +163,33 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
 
         this.isWebsite = true;
     },
+    /**
+     * @override
+     */
+    willStart: function () {
+        var self = this;
+        return this._super().then(function (el) {
```

Comment:
No need of `willStart` here, all of this should be in the `start` method. `willStart` is used to preload data the rendering needs, you do not need to load any data here.